app/script/entity/Conversation.js

```diff
@@ -193,6 +193,12 @@ z.entity.Conversation = class Conversation {
       }).length;
     });
 
+    this.unreadSelfMentionsCount = ko.pureComputed(() => {
+      return this.unreadEvents().filter(messageEntity => {
+        return messageEntity.is_content() && messageEntity.isUserMentioned(this.self.id);
+      }).length;
+    });
```

Comment:
This is all of computational overhead. Let's use the unread events observable and set a flag instead of iterating of the array again for every new message. We don't need the number anyway. A boolean will do.